src/composables/multisigTransactions.ts

```diff
@@ -112,25 +111,43 @@ export function useMultisigTransactions({ store }: UseMultisigTransactionsOption
     (await sdk.api.getTopHeader())?.height + MULTISIG_TRANSACTION_EXPIRATION_HEIGHT
     );
     const gaContractRpc = await sdk.getContractInstance({
-      aci: SimpleGAMultiSigAci,
+      source: multisigContract,
       contractAddress: contractId,
     });
 
-    await gaContractRpc.methods.confirm.send(spendTxHash, {
+    const result = await gaContractRpc.methods.confirm.send(spendTxHash, {
       FixedTTL: [expirationHeight],
     });
+
+    return result;
   }
 
-  async function revokeTx(spendTxHash: string, contractId: string) {
+  async function revokeTx(contractId: string, spendTxHash: string) {
     const sdk = await getSdk();
     const gaContractRpc = await sdk.getContractInstance({
-      aci: SimpleGAMultiSigAci,
+      source: multisigContract,
       contractAddress: contractId,
     });
 
-    await gaContractRpc.methods.revoke.send(
+    const result = await gaContractRpc.methods.revoke.send(
       Uint8Array.from(Buffer.from(spendTxHash, 'hex')),
     );
+
+    return result;
+  }
+
+  async function refuseTx(contractId: string, spendTxHash: string) {
```

Comment:
It would be awesome if you could add some description what this method does and what is the difference between this and the `revokeTx`.